superset/connectors/druid/models.py

```diff
@@ -898,7 +898,7 @@ def run_query(  # noqa / druid
             row_limit=None,
             inner_from_dttm=None, inner_to_dttm=None,
             orderby=None,
-            extras=None,  # noqa
+            extras={},  # noqa
```

Comment:
this is a python gotcha:
http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments,
just set it to `None` here and go with `extras = extras or {}` in the function body

Reply:
👍